utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -330,9 +330,29 @@ function Set-TemplateReferencesSection {
     $TextInfo = (Get-Culture).TextInfo
     foreach ($resourceType in $relevantResourceTypes) {
         $Type, $Resource = $resourceType.Type -split '/', 2
-        $sectionContent += ('- [{0}](https://docs.microsoft.com/en-us/azure/templates/{1}/{2}/{3})' -f $TextInfo.ToTitleCase($Resource), $Type, $resourceType.ApiVersion, $Resource)
+        $ResourceReferenceTitle = $TextInfo.ToTitleCase($Resource)
+        # Validate if Reference URL Is working
+        $TemplatesBaseUrl = 'https://docs.microsoft.com/en-us/azure/templates'
+        try {
+            $ResourceReferenceUrl = '{0}/{1}/{2}/{3}' -f $TemplatesBaseUrl, $Type, $resourceType.ApiVersion, $Resource
+            Invoke-WebRequest -Uri $ResourceReferenceUrl | Out-Null
```

Comment:
Change to `$null = ` instead of `Out-Null`.

Is used to do that too until somebody told me it is less performant

Reply:
good one. Did read about it and you're right. Thanks for the tip. Updated :)